extensions/change-case/src/change-case.tsx

```diff
@@ -52,12 +52,15 @@ async function readContent(preferredSource: string) {
 }
 
 function modifyCasesWrapper(input: string, case_: CaseFunction) {
-  const modifiedArr: string[] = [];
+  const modifiedRawArr: string[] = [];
+  const modifiedMarkddownArr: string[] = [];
   const lines = input.split("\n");
   for (const line of lines) {
-    modifiedArr.push(case_(line));
+    const modified = case_(line);
+    modifiedRawArr.push(modified);
+    modifiedMarkddownArr.push(modified + "\n");
```

Comment:
Also, should there be a case to handle empty lines? eg:
```js
modifiedMarkdownArr.push((modified.length === 0 ? "\u200B" : modified) + "\n");
```

Reply:
@erics118 
thanks you, it does make sense.
plz check my update.
I modified by checking the line directly instead of checking what was modified.